src/Microsoft.Data.SqlClient/netfx/ref/Microsoft.Data.SqlClient.csproj

```diff
@@ -16,6 +16,7 @@
     <Reference Include="System.Transactions" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="8.0.0" />
```

Comment:
Add version numbers to Versions.prop file

Reply:
Applied suggestion to use version in the Versions.prop file.